pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -239,6 +239,25 @@ lgi = buildLuarocksPackage {
     };
   };
 };
+lpeg = buildLuarocksPackage {
+  pname = "lpeg";
+  version = "1.0.2-1";
+
+  src = fetchurl {
+    url    = https://luarocks.org/lpeg-1.0.2-1.src.rock;
+    sha256 = "1g5zmfh0x7drc6mg2n0vvlga2hdc08cyp3hnb22mh1kzi63xdl70";
+  };
+  disabled = (luaOlder "5.1");
+  propagatedBuildInputs = [ lua ];
+
+  meta = {
+    homepage = "http://www.inf.puc-rio.br/~roberto/lpeg.html";
```

Comment:
The correct homepage is our previous one ending in `/` instead of `.html`, this one redirects to a 404 page. A minor annoyance that we're inheriting this wrong homepage link from the luarocks package now, if you ask me.

Dunno where you'd have to report that to get it fixed, I don't see anything helpful on the webpage.
https://luarocks.org/modules/gvvaughan/lpeg

Reply:
good catch. I sent him a mail. let's see. Hopefully the correct url can be picked up in the next update.